### PR TITLE
Add Nexus related SDK metrics

### PR DESCRIPTION
**Pull request by bergundy**
Modified the sdk-metrics page to show Nexus related metrics.

Note that Java SDK is still not released but I figured it can't hurt to already include this information.
@Quinn-With-Two-Ns to review for Java.

Also fixed an error in the tags for `activity_poll_no_task`, `activity_type` isn't available there.

### CODE REVIEW

File: docs/references/sdk-metrics.mdx [hunk 1]
```diff
@@ -452,3 +459,49 @@ The Schedule-To-Start time of a Workflow Task.
...
+
+### nexus_task_schedule_to_start_latency
+
+The Schedule-To-Start time of a Nexus Task in seconds. The schedule time is taken from when the corresponding request
+hit the Frontend service to when the SDK started processing the task.
+
+This time is limited by the `Request-Timeout` header given to the Frontend when handling this request.
+
+- Type: Histogram
+- Available in: Core, Go, Java
```
Comment (bergundy) [inline docs/references/sdk-metrics.mdx:479]:
```suggestion
- Available in: Go, Java
```

File: docs/references/sdk-metrics.mdx [hunk 2]
```diff
@@ -452,3 +459,49 @@ The Schedule-To-Start time of a Workflow Task.
...
+- Tags: `namespace`, `task_queue`, `nexus_service`, `nexus_operation`
+
+### nexus_task_execution_failed
+
+Handling of a Nexus Task resulted in a failed result. Does not take into account unsuccessful Operation results when
+starting an Operation.
```
Comment (bergundy) [inline docs/references/sdk-metrics.mdx:485]: We may want to fix this. Seems somewhat inconsistent.

Comment (bergundy) [reply docs/references/sdk-metrics.mdx:485]: What do we do in Java there?

Comment (Quinn-With-Two-Ns) [reply docs/references/sdk-metrics.mdx:485]: Sorry, what do we want to fix?

Comment (Quinn-With-Two-Ns) [reply docs/references/sdk-metrics.mdx:485]: This is accurate for Java

Comment (Quinn-With-Two-Ns) [reply docs/references/sdk-metrics.mdx:485]: Yeah I could see that , I think that is more consistent with what we do for activities

Comment (bergundy) [reply docs/references/sdk-metrics.mdx:485]: PR to change Go: https://github.com/temporalio/sdk-go/pull/1664

Comment (bergundy) [reply docs/references/sdk-metrics.mdx:485]: I'll fix the comment here.

Comment (bergundy) [reply docs/references/sdk-metrics.mdx]: Done.

**Review by jsundai — APPROVED** [review]

**Pull request merged**